docs/topics/genericgateway.md

```diff
@@ -47,18 +64,60 @@ When calling the Generic Gateway `simpleevent` endpoint, you must include a cust
 }
 ```
 
-`Ref` and `commit` values are used to configure the specific revision that Brigade will pull from your repository, if you're using one. If both values are missing, then your Build's `ref` will be set to `master`. If you do not wish to provide any payload, just include an empty JSON object (like `{}`).
+`ref` and `commit` values are used to configure the specific revision that Brigade will pull from your repository, if you're using one. If both values are missing, your Build's `ref` will be set to `master`. Of course, you can also provide any other values you need.
```

Comment:
As this gateway would be mostly used for non-Git scenarios, would it make sense to add this under a note?